src/auth/useAuth.ts

```diff
@@ -0,0 +1,77 @@
+import { useEffect, useCallback, useState } from 'react';
+import { useDispatch } from 'react-redux';
+import {
+  useApiTokens,
+  useOidcClient,
+  useApiTokensClientTracking,
+  isApiTokensUpdatedSignal,
+  isApiTokensRemovedSignal,
+  isApiTokensRenewalStartedSignal,
+  useAuthenticatedUser
+} from 'hds-react';
+import { setRedirectUrlToSessionStorage } from '@/util/storage';
+import { Routes, getRouteById } from '@/root/routes';
+import { clearApiToken, clearUser, userFound, receiveApiToken } from './actions';
+
+
+const useAuth = () => {
+  const [loggedIn, setLoggedIn] = useState(false);
+  const { login: oidcLogin, logout: oidcLogout, isRenewing: oidcIsRenewing } = useOidcClient();
+  const authenticatedUser = useAuthenticatedUser();
+  const dispatch = useDispatch();
+  const [apiTokensClientSignal, apiTokensClientSignalReset] = useApiTokensClientTracking();
+  const { getStoredApiTokens } = useApiTokens();
+
+  const setLoggedInIfApiTokenExists = useCallback(() => {
+    const [_error, apiToken] = getStoredApiTokens();
```

Comment:
Consider logging possible error

Reply:
Normally _error is null, so if we want to log possible errors from getStoreApiTokens, it could be logged inside an `if (_error) {}` block. Might help debugging, but not necessary.